src/PackagePrivate/Humanizer/InternationalizedHumanizer.php

```diff
@@ -157,7 +168,14 @@ private function humanizeYear( int $year, UnspecifiedDigit $unspecifiedDigit ):
     {
 	    $endingChar = $this->needsYearEndingChar($unspecifiedDigit) ? 's' : '';
 
-		return $year >= 0 ? (string)$year . $endingChar : (string)(-$year) . $this->message('edtf-bc');
+	    $humanized = '';
+	    if (abs($year) < 1000) {
+	        $humanized .= $this->message('edtf-year');
+        }
+
+		return $year >= 0
+            ? $humanized . (string)$year . $endingChar
+            : $humanized . (string)(-$year) . $this->message('edtf-bc');
```

Comment:
I'm pretty sure the translation people will not be happy with what we did here. Or with the space in `edtf-year`.

Instead can have 4 messages:

* BC
* AC
* BC < 1000
* AC < 1000

Then we don't have to play lego, don't hardcode ordering in the logic here, and avoid partial formatting in messages.

Reply:
```php
if ( year <= -1000 ) {
    return $this->message( 'edtf-bc-year', $year );
}

if ( year < 0 ) {
    return $this->message( 'edtf-bc-year-short', $year );
}

if ( year < 1000 ) {
    return $this->message( 'edtf-year-short', $year );
}

 return $this->message( 'edtf-year-long', $year );
```